front/src/modules/scenario/components/ScenarioExplorer/ScenarioExplorer.tsx

```diff
@@ -33,6 +36,7 @@ const ScenarioExplorer = ({
   const stdcmV2Activated = useSelector(getStdcmV2Activated);
 
   const { updateInfraID, updateTimetableID, updateScenarioID } = useOsrdConfActions();
+  const { updateStdcmEnvironment } = useOsrdConfActions() as StdcmConfSliceActions;
```

Comment:
Can be merge in a single instruction
```suggestion
  const { updateInfraID, updateTimetableID, updateScenarioID, updateStdcmEnvironment } = useOsrdConfActions() as StdcmConfSliceActions;
```